images/hooks/werf.inc.yaml

```diff
@@ -0,0 +1,30 @@
+---
+image: {{ $.ImageName }}
+final: false
+from: {{ .Images.BASE_GOLANG_23_BOOKWORM }}
+git:
+- add: /images/{{ $.ImageName }}
+  to: /app/images/hooks
+  stageDependencies:
+    install:
+    - go.mod
+    - go.sum
+    setup:
+    - "**/*.go"
+mount:
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
+shell:
+  install:
+  - cd /app/images/hooks
+  - go mod download
+  setup:
+  - cd /app/images/hooks
+  - export GOOS=linux
+  - export GOARCH=amd64
+  - export CGO_ENABLED=0
+  - go build -v -a -o /hooks/000-ca-discovery ./cmd/000-ca-discovery
```

Comment:
`-a` slows down sequential builds, this flag is for rebuilding every dependency.
`-v` is not useful for diagnostic, it just adds noise to the build log.

Use these flags from virt-artifact to reduce binary size (strip and no debug symbols):
```
go build -ldflags="-s -w" -o /kubevirt-binaries/virt-exportproxy ./cmd/virt-exportproxy/
```